helm/ffc-demo-payment-service/values.yaml

```diff
@@ -70,4 +70,8 @@ livenessProbe:
   periodSeconds: 10
   failureThreshold: 3
 
-aadPodIdentity: true
+aadPodIdentity: false 
```

Comment:
Can we just kill off podIdentity reference?

Reply:
@DaRoszko Done